features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/des/des_alt.c

```diff
@@ -342,66 +361,71 @@ static int mbedtls_des_docrypt(uint16_t keyopt, uint8_t key[3][MBEDTLS_DES_KEY_S
     *tdes_key ++ = val;
     val = nu_get32_be(key[2] + 4);
     *tdes_key ++ = val;
-    
```

Comment:
Similar question here: Why not performing the swapping upfront and then calling `TDES_SetKey`?

Reply:
@hanno-arm To call `TDES_SetKey`, I need to make up a 2-dimension array to pass keys, so DES key registers are directly configured. I corrected the comment in https://github.com/ARMmbed/mbed-os/pull/4925/commits/e2452a51e9e23b97fb85ac7caf86f62ec380465b.

Reply:
@ccli8 Yes, it would involve a slight memory overhead, at the benefit of unifying some code.
With such duplications of driver-code it's not easy to track which places to modify when the driver needs to be changed at some point. But we can wait for another opinion on that.

Reply:
@hanno-arm Same as above. I changed it by calling BSP driver in https://github.com/ARMmbed/mbed-os/pull/4925/commits/5f229a3d33096787ad8a312bb561159a432fb7c0.